administrative-sdk/choice-challenge/choice-challenge.js

```diff
@@ -49,10 +52,25 @@ export default class ChoiceChallenge {
      * @type {string} [question] A hint or question related to the choices.
      */
     this.question = question;
-    if (typeof choices !== 'object') {
+
+    if (!Array.isArray(choices)) {
```

Comment:
I think we need to decide. Either this checking could stay, but calls like listChoiceRecognitions should accept a challenge id and organisation id field instead of a ChoiceChallenge object that's filled with dummy data (which user is now required to do).
I assume same goes for Speech and Analysis.